src/test/e2e/swagger/swagger-section/update-by-id.spec.ts

```
@@ -0,0 +1,279 @@
+    await expect(elements.downloadButton).toBeVisible();
+    await expect(elements.downloadButton).toBeEnabled();
+
+    const [download] = await Promise.all([
+      page.waitForEvent('download'),
+      elements.downloadButton.click(),
+    ]);
+
+    const filePath: string = await download.path();
+    expect(filePath).not.toBeNull();
+
+    const buffer: Buffer<ArrayBufferLike> = await fs.readFile(filePath!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)